src/Pact/Types/Exp.hs

```diff
@@ -196,18 +214,24 @@ instance Pretty Separator where
   pretty ColonEquals = ":="
   pretty Comma = ","
 instance ToJSON Separator where
-  toJSON Colon = ":"
-  toJSON ColonEquals = ":="
-  toJSON Comma = ","
+  toJSON Colon = enableToJSON "Pact.Types.Exp.Separator" $ ":"
+  toJSON ColonEquals = enableToJSON "Pact.Types.Exp.Separator" $ ":="
+  toJSON Comma = enableToJSON "Pact.Types.Exp.Separator" $ ","
+  toEncoding Colon = toEncoding @String":"
```

Comment:
What does it mean that there is no space after `String` here?

Reply:
I hope that it just means that I forgot to add a space, but that it doesn't make a difference :-)